test/test_reliability_icc.rb

```diff
@@ -182,8 +182,8 @@ class StatsampleReliabilityIccTestCase < Minitest::Test
             should 'bounds be equal' do
               @icc.type = t
               @r_icc = @iccs[t.to_s]
-              assert_in_delta(@r_icc['lbound'], @icc.lbound)
-              assert_in_delta(@r_icc['ubound'], @icc.ubound)
+              assert_in_delta(@r_icc['lbound'], @icc.lbound, 0.1)
```

Comment:
The default delta is 0.001 ([according to the docs](http://apidock.com/ruby/MiniTest/Assertions/assert_in_delta)). Did you test with `delta = 0.01` as well? Or is it only passing with 0.1?